src/components/calcite-shell-panel/calcite-shell-panel.scss

```diff
@@ -99,12 +81,32 @@
   @apply hidden;
 }
 
-:host([position="start"]) slot[name="action-bar"]::slotted(calcite-action-bar) {
-  @apply border-r border-r-color-3;
-  border-right-style: solid;
+slot[name="action-bar"]::slotted(calcite-action-bar),
+.content ::slotted(calcite-flow),
+.content ::slotted(calcite-panel) {
+  @apply border border-color-3 border-solid;
+}
+
+:host([position="start"]) slot[name="action-bar"]::slotted(calcite-action-bar),
+:host([position="start"]) .content ::slotted(calcite-flow),
+:host([position="start"]) .content ::slotted(calcite-panel) {
+  @apply border-l-0;
+}
+
+:host([position="end"]) slot[name="action-bar"]::slotted(calcite-action-bar),
+:host([position="end"]) .content ::slotted(calcite-flow),
+:host([position="end"]) .content ::slotted(calcite-panel) {
+  @apply border-r-0;
+}
+
+:host([position="start"]) slot[name="action-bar"]::slotted(calcite-action-bar),
+:host([position="start"]) .calcite--rtl .content ::slotted(calcite-flow),
+:host([position="start"]) .calcite--rtl .content ::slotted(calcite-panel) {
+  @apply border-l border-r-0;
```

Comment:
Instead of using the `.calcite--rtl` class, can we use CSS logical properties here? for border-left and border-right? https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties

Reply:
Ah...good call.
For some reason I thought those weren't supported.
Will do.